spec/Section 7 -- Response.md

```diff
@@ -60,33 +61,34 @@ response must not be empty. It must contain at least one error. The errors it
 contains should indicate why no data was able to be returned.
 
 If the `data` entry in the response is present (including if it is the value
-{null}), the `errors` entry in the response may contain any field errors that
-were raised during execution. If field errors were raised during execution, it
-should contain those errors.
+{null}), the `errors` entry in the response may contain a list of all _field
+error_ that were raised during execution. If any _field error_ was raised during
+execution, it should contain the `errors` entry.
```

Comment:
Because we've gone from plural to singular, the "it" has become ambiguous (it could be referring to the "field error"); so let's clarify:

```suggestion
execution, the request should contain the `errors` entry.
```

Reply:
I believe the second sentence (If any field error was raised... it SHOULD (?) contain ...) is a repetition of the first sentence (the errors entry in the response MAY(?) contain a list...)  - basically saying 'errors' contains errors raised. 
At the same time, there is slight contradiction - MAY vs SHOULD

Reply:
I suggest to remove the second sentence and change MAY -> SHOULD in the first.

Reply:
Lots of confusing interchanging of must/should - let me try to reword this section further to clarify